src/protocol/commit_strategy.md

```diff
@@ -1,21 +1,24 @@
 
 ## Cost
-These strategies mainly differ in the number of portions committed (`K`). As `K` increases, the major cost that increases is the size of the artefact generated (*artefact size* in the table above). The artefact generated, as well as the factor by which their associated size increases when `K` increases (*size scaling*) are detailed in the table below.
+The commitment strategies differ mainly in the number of committed ranges (`K`). As `K` increases, the primary cost is the size of the generated artefact. The table below details the artefacts and how their sizes scale with `K`.
 
-| # | Artefact | Description | Size Scaling | Explanation |
-| - | -------- | ----------- | ------------ | ---------------------- |
-| 1 | `Attestation` | Artifact signed by the `Notary` attesting to the authenticity of the plaintext from a TLS session | Constant | `Attestation` only contains data  that remains constant-sized regardless of `K`, e.g. the merkle root of the commitments |
-| 2 | `Secret` | Artifact containing secret data that correspond to commitments in `Attestation` | Linear | `Secret` contains some data whose sizes scale linearly with `K`, e.g. a merkle tree whose number of leaves equals to `K` |
+| Artefact      | Description                                                                                       | Size Scaling | Explanation                                                                                                              |
+| ------------- | ------------------------------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------ |
+| `Attestation` | Artifact signed by the `Notary` attesting to the authenticity of the plaintext from a TLS session | Constant     | `Attestation` only contains data that remains constant-sized regardless of `K`, e.g., the Merkle root of the commitments |
+| `Secret`      | Artifact containing secret data that correspond to commitments in `Attestation`                   | Linear       | `Secret` contains some data whose sizes scale linearly with `K`, e.g., a Merkle tree whose number of leaves equals `K`   |
 
-Using the default hash algorithm (i.e. BLAKE3), every additional portion committed costs around 250 bytes of increment in the size of `Secret`. For more details, please visit this [Jupyter notebook](https://colab.research.google.com/drive/1o7IOwxZ9DuZLNsg6sQKp2y25kzgUPJhC?usp=sharing).
+Using the default hash algorithm (i.e., BLAKE3), every additional range committed costs around 250 bytes of increment in the size of `Secret`. For more details, please visit this [Jupyter notebook](https://colab.research.google.com/drive/1o7IOwxZ9DuZLNsg6sQKp2y25kzgUPJhC?usp=sharing).
```

Comment:
extra comma?

Reply:
In US english, there should be a comma indeed (different in UK).